.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+
+If you are unable to find any open GitHub issues addressing the problem you found, your next step will be to [open a new one][2] using the respective 'Bug' template.
+
+Your issue report should contain a title and as much relevant information as possible of the issue. Your goal should be to make it easy for yourself - and others - to reproduce the bug and figure out a fix.
+
+### Requesting a new feature
+
+If there's a new feature that you want to see added to CKAN, you can [open an issue][2] using the respective 'Feature' template, or you can write the code yourself.
+
+## Helping with issues
+### Verifying issues
+
+For starters, it helps just to verify bug reports. Can you reproduce the reported issue on your own computer? If so, you can add a comment to the issue saying that you're seeing the same thing.
+
+If an issue is very vague, can you help narrow it down to something more specific? Maybe you can provide additional information to help reproduce a bug, or help by eliminating needless steps that aren't required to demonstrate the problem.
+
+### Testing patches
+
+You can also help out by examining pull requests that have been submitted to CKAN via GitHub. In order to apply someone's changes, you need to first create a dedicated branch:
+
+`$ git checkout -b testing_branch`
+
+Then, you can use their remote branch to update your codebase. For example, let's say the GitHub user JohnSmith has forked and pushed to a topic branch "orange" located at https://github.com/JohnSmith/CKAN.
+
+```
+$ git remote add JohnSmith https://github.com/JohnSmith/CKAN.git
+$ git pull JohnSmith orange
+```
+
+After applying their branch, test it out! Comment on the GitHub issue to apply some changes or by giving your approval.
+
+## Translating
+
+We recently started working on translating CKAN and are happy to have people volunteer to translate with us. Just follow these steps:
+
+* Fork the repository:
+
+```
+$ git clone https://github.com/KSP-CKAN/CKAN.git
+$ cd CKAN
+$ git checkout -b new_branch
+``` 
+
+* Add a folder for your own language, for example: `GUI/it-IT` for Italian.
+
+* Copy the contents of `GUI/en-US` into your own language folder.
+
+* Rename the files from `en-US` to `it-IT`.
+
+* Start translating the files.
+
+**Translations we already have:**
+
+* English (US)
+* English (AU)
+* German
+
+## Creating pull requests
+
+Before you make a pull request, you have to agree to contribute under these licenses:
+
+* Contributions to the CKAN source *must* be licensed under the [MIT license][3].
+
+* Contributions of CKAN metadata files (.netkan or .ckan) *must* be be contributed under the [CC-0][4] license.
+
+### Creating a patch
+
+* Fork the repository:
+
+```
+$ git clone https://github.com/KSP-CKAN/CKAN.git
+$ cd CKAN
+$ git checkout -b new_branch
+``` 
+
+* Now get busy and add/edit code. Make sure to follow our [Coding conventions](#coding-conventions).
+
+* Commit your changes:
+
+```
+$ git commit -a
+$ git push
```

Comment:
Should be `git push -u origin new_branch`I think.
Also, can anyone create a branch in the CKAN repo?
Else you should clone the own fork from GitHub, not the main repo.